app/views/NewMessageView.tsx

```diff
@@ -313,7 +301,7 @@ class NewMessageView extends React.Component<INewMessageViewProps, INewMessageVi
 			<FlatList
 				data={search.length > 0 ? search : chats}
 				extraData={this.state}
-				keyExtractor={item => item._id}
+				keyExtractor={item => item._id || item.rid}
```

Comment:
Now with the code more readable, I think we can remove that `_id`, can't we?

Reply:
Worse than not, when the object comes directly from the api it doesn't have the rid.